OloEngine/tests/Networking/MMOOptimizationTest.cpp

```
@@ -0,0 +1,149 @@
+{
+    InstancePortalComponent portal;
+    EXPECT_EQ(portal.TargetZoneID, 0u);
+    EXPECT_EQ(portal.InstanceType, 0u);
+    EXPECT_EQ(portal.MaxPlayers, 5u);
+}
+
+// ============================================================================
+// Crowd Update Threshold Test
+// ============================================================================
+
+TEST(MMOOptimization, CrowdUpdateThreshold)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)